pyspark_ai/pyspark_ai.py

```diff
@@ -62,7 +67,7 @@ def __init__(
         cache_file_location: Optional[str] = None,
         vector_store_dir: Optional[str] = None,
         vector_store_max_gb: Optional[float] = 16,
-        encoding: Optional[Encoding] = None,
+        encoding: "Optional[Encoding]" = None,
```

Comment:
I suspect if anyone specifies this option...How about let's just remove it and mention it in the PR description.